src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/LoginController.java

```diff
@@ -58,7 +58,7 @@ public ResponseEntity<String> login(HttpServletRequest httpRequest) {
             SecurityContextHolder.getContext()
                     .setAuthentication(new UsernamePasswordAuthenticationToken(
                             new AdminSession(this.grantAdminId, this.funderId, "Test", "User", this.funderName,
-                                    emailAddress),
+                                    emailAddress, null),
```

Comment:
If we've signed in locally wouldn't we want valid roles here 👀

Reply:
Discussed offline - going to remove this block previously used for skipping cola auth locally